packages/quantic/cypress/e2e/default-1/sort/sort.cypress.ts

```diff
@@ -32,15 +37,58 @@ describe('quantic-sort', () => {
     },
   ];
   const sortOptionValues = sortOptions.map((option) => option.value);
+  const defaultCustomOptionValue = 'date ascending';
+
+  const customSortOptions = [
+    {
+      label: 'Date ascending',
+      value: 'date ascending',
+      criterion: {
+        by: 'date',
+        order: 'ascending',
+      },
+    },
+    {
+      label: 'Views Descending',
+      value: '@ytviewcount descending',
+      criterion: {
+        by: 'field',
+        field: 'ytviewcount',
+        order: 'descending',
+      },
+    },
+    {
+      label: 'No Sort',
+      value: 'nosort',
+    },
+  ];
 
+  const customSortOptionsValues = customSortOptions.map(
+    (option) => option.value
+  );
```

Comment:
I applied the feedback for this, but for the filtering of the array in the test, I cant remove it because the first option is selected by default and clicking on a selected item doesnt re-trigger requests (search/analytics) so the test fails.